lib/src/lib/table/components/table-cell/table-cell.component.html

```diff
@@ -11,27 +11,33 @@
     [matTooltipPosition]="cell().cellHeaderTooltipPosition || 'above'"
     [style.width]="columnWidth()"
     [style.max-width]="columnMaxWidth()">
-    <ng-container
-      *ngComponentOutlet="
-        cell().headerRenderer || defaultHeaderRenderer;
-        inputs: { columnConfig: cell(), disableSort: disableSort() }
-      " />
+    @defer (on viewport) {
```

Comment:
suggestion: try it with a second argument:
 `@defer (on viewport; on idle) {`
that way, when the browser is idle, it will start rendering the off-screen cells, making scrolling more performant later.
but it really needs to be tried (also in stip) to see if it's actually better/impactful

Reply:
on idle looks to prolong the render time again. I've added a prefetch on idle